webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7313,7 +7359,28 @@ var CURRENT_GROUP_DATA={};
                 };
                 return conversationDetail;
             };
-
+            _this.getBotDetails = function () {
```

Comment:
Give a proper name, use get when the function is returning something, here this function is not returning anything

Reply:
done

Reply:
Please done only when it's actually 'done' so that other people come for review only it's really 'done'. It will save everyone's time

Reply:
okay

Reply:
resolved in new commit, please check